ls1intum/Artemis
# `Assessment`: Show point deduction directly in the text feedback component

### Checklist
#### General


#### Client

### Motivation and Context


Fixes #5114 

### Description

Added the points assigned to a text feedback to the beginning of the associated feedback.
Renamed `credits` to `points` in the English translation. 
In addition, the points are now also getting localized based on the set language in Artemis.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Text Exercise with Feedback

1. Submit a text as a student
2. Assess the submission and use a negative as well as positive inline feedback
3. As a student, you should see the points directly on the right side of the inline feedback
4. Change the language and make sure the points get localized as well

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
| text-result.component.ts | 70.73% | 95.16% |
| additional-feedback.component.ts | 78.57% | 100% |

### Screenshots



![Bildschirmfoto 2022-05-20 um 23 04 23](https://user-images.githubusercontent.com/9954674/169610990-0b1b6fea-bef5-429f-8bf8-a8b711d0c72d.png)

![Bildschirmfoto 2022-05-20 um 23 03 57](https://user-images.githubusercontent.com/9954674/169610995-3ee9ec17-409f-4ab5-bc29-04dc5ff37271.png)
